pages/common/arp.md

```diff
@@ -2,11 +2,15 @@
 
 > Show and manipulate your system's ARP cache.
 
-- Show current arp table:
+- Show current arp table [as numeric address]:
```

Comment:
We don't do "optionals" or parameter describing here even though it's an easy way to introduce two options at the same time. Drop the brackets, with the added danger that the options below aren't clear individually.